File: c/hello.c
#include<stdio.h>
#include"max.h"


int main(){
	int a, b;
	a = 12;
	b = 34;
	int arr[5] = {
		5, 4, 6, 3, 8
	};
	int m = max(arr, 5);
	printf("max = %d\n", m);
	printf("%d\n", a + b);
	printf("Hello, C World!\n");
	return 0;
}
// gcc -S hello.c

File: py/hello.py
hello = input("Please input your name:\n")

print("Hello ")
print(hello)
print("\n")
if hello == "hello":
    print("Hello, Python world!\n")
else:
    print("Welcome to Python Program.\n")

print("The End")

inp = input("Please input a number:\n")
num = (int)(inp)
sums = 0

for i in range(1, num):
    sums += i;
print("The sums between 1 to ", num, " is ", sums)
